manifests/init.pp

```diff
@@ -79,16 +79,14 @@
 #  }
 #
 define sslcertificate (
-  $password,
-  $location,
-  $thumbprint,
-  $root_store    = 'LocalMachine',
-  $store_dir     = 'My',
-  $scripts_dir   = 'C:\temp',
-  $exportable = true) {
-  validate_re($name, '^(.)+$', "Must pass name to ${module_name}[${title}]")
```

Comment:
I'm pretty sure this is some very very legacy foo. I saw this in multiple modules in the past weeks.